core/backend/src/CheckpointManager.ts

```diff
@@ -349,7 +352,10 @@ export class CheckpointManager {
           const accessToken = checkpoint.accessToken;
           const toIndex = checkpoint.changeset.index ??
             (await IModelHost.hubAccess.getChangesetFromVersion({ accessToken, iModelId: checkpoint.iModelId, version: IModelVersion.asOfChangeSet(checkpoint.changeset.id) })).index;
+          const stopwatch = new StopWatch(`[${checkpoint.changeset.id}]`, true);
+          Logger.logInfo(loggerCategory, `Starting application of changeset with id ${stopwatch.description}`);
           await BriefcaseManager.pullAndApplyChangesets(db, { accessToken, toIndex });
+          Logger.logInfo(loggerCategory, `Applied changeset with id ${stopwatch.description} (${stopwatch.elapsedSeconds} seconds)`);
```

Comment:
Since there can be multiple changesets to apply, this timer could be misleading. I think we could apply 100 changesets and we would get the impression that the one singular changeset id that you're logging took the time to apply 100 changesets when it would potentially only account for 1/100th of that time.

Maybe we should log each individual changeset apply time? Which would be inside of pullAndApplychangesets itself ?